SU2_CFD/src/iteration_structure.cpp

```diff
@@ -1464,17 +1412,16 @@ void CFEAIteration::Update(COutput *output,
 
     /*--- Verify convergence criteria (based on total time) ---*/
 
-    Physical_dt = config[val_iZone]->GetDelta_DynTime();
-    Physical_t  = (TimeIter+1)*Physical_dt;
-    if (Physical_t >=  config[val_iZone]->GetTotal_DynTime())
+    su2double Physical_dt = config[val_iZone]->GetDelta_DynTime();
```

Comment:
const for these two?

Reply:
My criteria lately has been, if the variable is used right away no need for const, but I can change it.

Reply:
Prob best to just do it always... makes the intent clear for others than come after you and may move things around or reuse